engines/director/lingo/lingo-builtins.cpp

```diff
@@ -1111,9 +1111,13 @@ void LB::b_getNthFileNameInFolder(int nargs) {
 }
 
 void LB::b_open(int nargs) {
-	g_lingo->printSTUBWithArglist("b_open", nargs);
-
-	g_lingo->dropStack(nargs);
+	Datum d = g_lingo->pop();
+	if (nargs == 2)
+		g_lingo->pop();
+	warning("LB::b_open(): Unsupported command open encountered -> The movie tried to open %s", d.asString().c_str());
+	Common::U32String message = _("Unsupported command open encountered -> The movie tried to execute open lingo command");
```

Comment:
Please put '%s' in the message as well, so the user sees what was attempted to open, e.g.:

```C++
Common::U32String message = Common::String::format(_("Unsupported command open encountered -> The movie tried to execute '%s' command", d.asString().c_str());
```